src/Sentry.Unity/ScriptableSentryUnityOptions.cs

```diff
@@ -167,7 +167,7 @@ internal SentryUnityOptions ToSentryUnityOptions(bool isBuilding, ISentryUnityIn
             OptionsConfiguration?.Configure(options);
 
             // Doing this after the configure callback to allow users to programmatically opt out
-            if (Il2CppLineNumberSupportEnabled)
+            if (!isBuilding && Il2CppLineNumberSupportEnabled)
```

Comment:
At build-time this will log a warning trying to attach the IL2CPP exception processor.